tests/Unit/DataStructures/Test_Variables.cpp

```diff
@@ -601,6 +634,28 @@ SPECTRE_TEST_CASE("Unit.DataStructures.Variables.add_slice_to_data",
         get<VariablesTestTags_detail::vector>(vars));
 }
 
+SPECTRE_TEST_CASE("Unit.DataStructures.Variables", "[DataStructures][Unit]"){
```

Comment:
Formatting `){` and 2 space indentation. Please try using `git clang-format -f` before committing. It will format only the lines you touched :)

Reply:
sorry, this one's rough because clang-format removes all indentation, getting confused from the `SECTION`s, so had to resort to (fallible) manual formatting .

Reply:
Ah, getting. I've encountered clang-format's dislike of `SECTION` too, but don't understand it. Understood what the struggle was :)